go-controller/pkg/ovn/master.go

```diff
@@ -480,6 +480,13 @@ func (oc *DefaultNetworkController) deleteStaleNodeChassis(node *kapi.Node) erro
 		p := func(item *sbdb.Chassis) bool {
 			return item.Name == staleChassis
 		}
+		if err = libovsdbops.DeleteChassisTemplateRecord(oc.nbClient, []*nbdb.ChassisTemplateVar{{Chassis: staleChassis}}...); err != nil {
```

Comment:
no need to pass `[]*nbdb.ChassisTemplateVar`, you can pass a single *nbdb.ChassisTemplateVar. similar in other places in this file.

Reply:
True, I fixed it.